search/search_repository.go

```diff
@@ -266,6 +266,10 @@ func parseMap(queryMap map[string]interface{}, q *Query) {
 		case bool:
 			s := concreteVal
 			q.Negate = s
+		case nil:
+			q.Name = key
+			var s *string
+			q.Value = s
```

Comment:
@baijum this works if you pass nil

Reply:
Fixed in 3311591f05dd3cc468ced7f648277870497e1c5f